Data/Model/BaseModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Model
+{
+    public class BaseModel
```

Comment:
Nunca vamos instanciar essa classe, deve ser marcada como abstract pois é usada apenas para extensão
```suggestion
    public abstract class BaseModel
```